### DISCUSSION TITLE
Adding additional db indexes or extending Prisma schema

### DISCUSSION THREAD

**Discussion by Dador** [keystonejs/keystone#6811] [Questions] [answered]
Seems like currently there is only option to add index for some columns using `isIndexed: true`.
How to add index with several columns?

Or is there a way to extend Prisma schema?

**Comment by bladey** [comment, marked as answer]
Hey @Dador,

We're using GitHub discussions for feature requests only, could you please ask this question in the Keystone Community Slack and we'll assist - https://community.keystonejs.com

Thank you!
